server/controllers/column.js

```diff
@@ -0,0 +1,58 @@
+const asyncHandler = require("express-async-handler");
+const Column = require("../models/Column");
+const { v4: uuidv4, v4 } = require("uuid");
+
+exports.getColumns = asyncHandler(async (req, res) => {
+  const Columns = await Column.find();
+
+  return res.json(Columns);
+});
+
+exports.getColumn = asyncHandler(async (req, res) => {
+  try {
+    const Column = await Column.findById(req.params.id);
+
+    if (!Column) res.status(404).json({ error: "Column not found" });
```

Comment:
The proper way with async handler is to set the status to 404 and then throw the error which will be handled by `express-async-handler`